src/InputQuery.php

```
@@ -372,4 +391,155 @@
+            return [];
+        }
+
+        $arrayData = $_FILES[$paramName];
+        
+        if (!is_array($arrayData)) {
+            return [];
+        }
+
+        // Check if this is HTML multiple file upload format
+        if (isset($arrayData['name']) && is_array($arrayData['name'])) {
+            return $this->convertMultipleFileFormat($arrayData);
```

PHPStan (2.1.15)

Parameter #1 $multipleFileData of method Ray\InputQuery\InputQuery<T of object>::convertMultipleFileFormat() expects array<string, mixed>, array given.


(argument.type)